brokers/vscode/broker_test.go

```diff
@@ -0,0 +1,220 @@
+//
+// Copyright (c) 2012-2018 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package vscode
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/eclipse/che-plugin-broker/model"
+)
+
+func Test_findAssetURL(t *testing.T) {
+	type args struct {
+	}
+	tests := []struct {
+		name     string
```

Comment:
As far as I see name field is not set anywhere but is used for `t.Run(tt.name, ...)`. Is it done by purpose?

Reply:
Yes, it is required by the Run function

Reply:
but it is always nil, right?. Can you just write `t.Run(nil, ...)`? Or consider setting meaningful name in your tests.

Reply:
ok